<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>970</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>NCEM Tomography Tool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>20</y>
      <width>271</width>
      <height>641</height>
     </rect>
    </property>
    <property name="title">
     <string>Tomography Series</string>
    </property>
    <widget class="QGroupBox" name="controlPanelBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>420</y>
       <width>151</width>
       <height>201</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Control Panel</string>
     </property>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>93</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QPushButton" name="btnMaxTilt">
           <property name="text">
            <string>Max째</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnMinTilt">
           <property name="text">
            <string>Min째</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnPreview_2">
           <property name="text">
            <string>Preview</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QPushButton" name="btnAcq">
           <property name="text">
            <string>Acquire</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAbo">
           <property name="text">
            <string>Abort</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="seriesBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>210</y>
       <width>151</width>
       <height>201</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Series Parameters</string>
     </property>
     <widget class="QWidget" name="formLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>131</width>
        <height>171</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::FieldsStayAtSizeHint</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="lblTim">
         <property name="text">
          <string>Exposure</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="expBox">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="maximum">
          <double>2.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblNum">
         <property name="text">
          <string>Num Tilts</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="numBox">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="minimum">
          <double>2.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblSte">
         <property name="text">
          <string>Tilt Step</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="steBox">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="minimum">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblMax">
         <property name="text">
          <string>Max Tilt</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="maxBox">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="suffix">
          <string>째</string>
         </property>
         <property name="minimum">
          <double>-70.000000000000000</double>
         </property>
         <property name="maximum">
          <double>80.000000000000000</double>
         </property>
         <property name="value">
          <double>80.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblMin">
         <property name="text">
          <string>Min Tilt</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="minBox">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="suffix">
          <string>째</string>
         </property>
         <property name="minimum">
          <double>-80.000000000000000</double>
         </property>
         <property name="maximum">
          <double>70.000000000000000</double>
         </property>
         <property name="value">
          <double>-80.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lblMin_2">
         <property name="text">
          <string>Repeats</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QDoubleSpinBox" name="repBox">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="suffix">
          <string/>
         </property>
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10.000000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="statusBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>151</width>
       <height>161</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Status</string>
     </property>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>131</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <property name="spacing">
        <number>-1</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Res:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblXYRes">
           <property name="text">
            <string>(res,res)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Shift:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblXYShift">
           <property name="text">
            <string>(짹x,짹y)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Delta Def:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblDeltaDefocus">
           <property name="text">
            <string>짹(def)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Defocus</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="defBox">
           <property name="suffix">
            <string>nm</string>
           </property>
           <property name="minimum">
            <double>-99.989999999999995</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>127</width>
        <height>21</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_17">
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QRadioButton" name="bin1">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>1</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="bin2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>2</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="bin4">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>4</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="lblBin">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>127</width>
        <height>21</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>Binning</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <zorder>verticalLayoutWidget_2</zorder>
     <zorder>verticalLayoutWidget</zorder>
     <zorder></zorder>
     <zorder></zorder>
     <zorder></zorder>
     <zorder></zorder>
     <zorder></zorder>
     <zorder>lblBin</zorder>
    </widget>
    <zorder>controlPanelBox</zorder>
    <zorder>seriesBox</zorder>
    <zorder>statusBox</zorder>
    <zorder></zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>900</y>
      <width>821</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Dataset</string>
    </property>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>481</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_2">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>20</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>670</y>
      <width>271</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Amender</string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>272</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QColumnView" name="columnView"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <item>
         <widget class="QPushButton" name="btnPreview">
          <property name="text">
           <string>Preview</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Go to...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnS2F">
          <property name="text">
           <string>&gt;&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnS2F_2">
          <property name="text">
           <string>&lt;&lt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Average</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDiscard">
          <property name="text">
           <string>Discard</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QColumnView" name="columnView_3"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>821</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>511</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_18">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_20">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_16">
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Time</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblTime_details">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_14">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Res:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblXYRes_details">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <item>
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Exposure:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblExp_details">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_21">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Alpha:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblAlpha_details">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Beta:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblBeta_details">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Defocus:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblDef_details">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>X:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblX_details">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Y:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblY_details">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Z:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblZ_details">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <zorder></zorder>
    <zorder></zorder>
    <zorder></zorder>
    <zorder></zorder>
    <zorder></zorder>
    <zorder></zorder>
    <zorder></zorder>
   </widget>
   <widget class="QGroupBox" name="plot_groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>820</width>
      <height>820</height>
     </rect>
    </property>
    <property name="title">
     <string>Image Viewer</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_19"/>
   </widget>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>lblBin</zorder>
   <zorder>splitter_2</zorder>
   <zorder>splitter</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>columnView</zorder>
   <zorder>columnView_3</zorder>
   <zorder>verticalLayoutWidget_3</zorder>
   <zorder></zorder>
   <zorder></zorder>
   <zorder></zorder>
   <zorder></zorder>
   <zorder></zorder>
   <zorder>bin2</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>plot_groupBox</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
